src/web/app/components/question-submission-form/question-submission-form.component.html

```diff
@@ -179,4 +179,4 @@
       <span *ngIf="model.recipientType === FeedbackParticipantType.STUDENTS">This question is for other students in this course and this course doesn't have any other student. Therefore, you will not be able to answer this question.</span>
     </div>
   </div>
-</div>
+</div>
```

Comment:
This should not be modified right?

Reply:
oh man - will revert this! Yes, it should not be modified